test-data/recipes/channel_sources/recipe.yaml

```diff
@@ -0,0 +1,7 @@
+package:
+  name: channel-sources
+  version: 0.1.0
+
+requirements:
+  build:
+    - rust ==1.88.0.dev20250428
```

Comment:
Hmmmmmm given the size of `rust` this might not be the fastest test :)

Can we render the recipe and inspect the channels that are in the JSON output and use that to make sure things work?

Reply:
<img width="652" alt="Screenshot 2025-04-30 at 11 30 25" src="https://github.com/user-attachments/assets/682e1a72-6a69-4b04-beba-912caa163af3" />

Reply:
[`0714037` (#1597)](https://github.com/prefix-dev/rattler-build/pull/1597/commits/0714037c7ddac33f0e6088518a89d0f6030c6c12)